src/test/java/com/nerodesk/om/aws/AwsFriendsTest.java

```diff
@@ -51,4 +56,30 @@ public void conformsToEqualsHashCodeContract() {
             .suppress(Warning.TRANSIENT_FIELDS)
             .verify();
     }
+
+    /**
+     * AwsFriends can split the friends list correctly.
+     * @throws IOException If something goes wrong.
+     * @todo 152:30min This test should actually verify that AwsFriends.names is
+     *  correct after removing/ejecting all friends - it should return an empty
+     *  list instead of a list with a single blank element. However, it is not
+     *  currently possible to unit test that functionality because of specific
+     *  calls to the AWS API done inside of method eject. Mocking with jcabi-s3
+     *  is also not an option because we can't control the metadata returned by
+     *  MkOcket. Once it is possible to test eject, change the test below to its
+     *  correct behavior described above.
+     */
+    @Test
+    public void splitsFriendCorrectly() throws IOException {
+        final String names = "";
```

Comment:
maybe it would be better to write more realistic test (it's it's possible) but ignore it?
Current implementation is rather useless.